cmd/destroy/remote.go

```diff
@@ -203,22 +206,26 @@ func (rd *remoteDestroyCommand) destroy(ctx context.Context, opts *Options) erro
 		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, okteto.GetContext().Namespace),
 		fmt.Sprintf("%s=%s", model.OktetoTokenEnvVar, okteto.GetContext().Token),
 		fmt.Sprintf("%s=%s", constants.OktetoTlsCertBase64EnvVar, base64.StdEncoding.EncodeToString(sc.Certificate)),
-		fmt.Sprintf("%s=%s", constants.OktetoInternalServerNameEnvVar, sc.ServerName),
```

Comment:
I think having the extra hosts implies that we can get rid off the internal server name flag on deploy and destroy. I think it is out of the scope of this PR but something to tackle soon, mentioning in case this is still relevant and some of the reviewers comments about it